constants.js

```diff
@@ -151,6 +156,8 @@ module.exports.ciFailureCheck = ciFailureCheck;
 module.exports.updateWithDevelopCheck = updateWithDevelopCheck;
 module.exports.respondToReviewCheck = respondToReviewCheck;
 module.exports.oldBuildLabelCheck = oldBuildLabelCheck;
+module.exports.ensureNewIssuesHaveProjectsCheck =
+ensureNewIssuesHaveProjectsCheck;
```

Comment:
```suggestion
module.exports.ensureNewIssuesHaveProjectsCheck = (
  ensureNewIssuesHaveProjectsCheck);
```

Reply:
Will change them in the latest commit Sir.Thanks.